src/cursor/cur_backup.c

```diff
@@ -789,7 +794,12 @@ __backup_start(
         WT_ERR(__wt_fopen(session, WT_LOGINCR_SRC, WT_FS_OPEN_CREATE, WT_STREAM_WRITE, &srcfs));
         WT_ERR(__backup_list_append(session, cb, dest));
     } else {
-        dest = WT_METADATA_BACKUP;
+        export_backup = F_ISSET(cb, WT_CURBACKUP_EXPORT);
+        if (export_backup)
```

Comment:
`export_backup`is only used once in the `if` statement above. We can remove the variable and use `F_ISSET` directly.

Reply:
I agree with this.